data/changelog.txt

```diff
@@ -1,3 +1,569 @@
+The introduction of background hobbies has added a new tab to character creation that allows the player to select one or more hobbies/backgrounds that flesh out what your character was doing prior to the Cataclysm in skills and traits.
+JSONized sidebar allows almost total control of what your sidebar looks like and what information it displays is up to you!
+Skill rust has been turned into a net gain for the player as any rusted xp now turns into an xp multiplier that helps your character learn faster.  As a result it now defaults to on.
+Passively boil water by putting dirty water into a fireproof container then placing the container on top of a fire.  Walk away and after an amount of time dependent on the amount of water to boil it will all be clean water!
+Monster weakpoints! Many monsters now have weakpoints that the player can hit to apply various effects and extra damage to the monster.  As the player dissects more corpses of monsters they can learn weakpoint proficiencies that increase the player's odds of hitting weakpoints.
+Characters now have cardio which will respond to your playstyle.  Go for a run every day and perform hard labor and your stamina will slowly go further and further.  Spend three months reading and performing low intensity crafts and your cardio will suffer.
+A half dozen new in repo mods: Innawoods for the survival, craft loop you crave. Backrooms which takes place in the neverending backrooms dimension, Massachusetts mod for people who desire geographic accuracy, Xedra Evolved a science fantasy mod with weird magic and vampirism, Tropicataclysm for your Cataclysm at a warmer latitude and Megafauna which like Dinoclysm targets an ancient epoch with creatures that outsize our modern equivalents.
+Take over a friendly NPC upon your death, if you've managed to recruit any NPCs to your faction you are given an option to take over one of them in the event of your character's death.
+Appliances and building based powergrids, allow you to set up various tools or even usable arcade machines in your home base.
+Two new Iso tilesets have been added to the game.
+Simple monster aggression differentation. Some monsters now know if another monster made them angry versus instead the PC.  If they get angry at other monsters they may not get angry at the PC.
+Vehicles now have solid walls even when angled which prevents monsters and fields from entering an angled vehicle.
+Dozens of updates and content additions to Dinomod, Aftershock and Magiclysm.
+Map memory is now unlimited, you will no longer forget places you have been before.
+There has been a distinct push to make many options and possibilities clearer to the player without having to search for them, UI that makes suggestions as to why the player might want to perform an action.
+
+## Statistics
```

Comment:
Start of the Statistics section that needs a python script to pull all this data.